examples/bakdata/atm-fraud-detection/config.yaml

```diff
@@ -11,3 +11,5 @@ schema_registry_url: "http://k8kafka-cp-schema-registry.kpops.svc.cluster.local:
 kafka_rest_host: "http://localhost:8082"
 
 kafka_connect_host: "http://localhost:8083"
+
+defaults_path: ./defaults
```

Comment:
if we're setting it in the config.yaml, we can remove it here https://github.com/bakdata/kpops/blob/main/docs/docs/user/examples/atm-fraud-pipeline.md

Reply:
good call btw 👍🏻

Reply:
Thanks, the schema says it is required in `config.yaml`. Does it get overridden if you provide a different path from the CLI?

Reply:
Fixed the example doc.

Reply:
Should I assign it to v1.0

Reply:
yes please. :) this bug needs to be fixed